diff --git a/sindy_rl/env.py b/sindy_rl/env.py
index 9f5cd43..8eae786 100644
--- a/sindy_rl/env.py
+++ b/sindy_rl/env.py
@@ -74,7 +74,7 @@ def rollout_env(env, policy, n_steps, n_steps_reset=np.inf, seed=None, verbose =
         if done or len(obs_list) > n_steps_reset:
             obs_list.pop(-1)
             trajs_obs.append(np.array(obs_list))
-            trajs_acts.append(np.array(act_list))
+            trajs_acts.append(np.array(act_list).reshape(-1, 1))
             trajs_rews.append(np.array(rew_list))
 
             obs_list = [safe_reset(env.reset())]
diff --git a/tutorial/dynamics.ipynb b/tutorial/dynamics.ipynb
index 23b9280..a155f31 100644
--- a/tutorial/dynamics.ipynb
+++ b/tutorial/dynamics.ipynb
@@ -1,3 +1,3 @@
 version https://git-lfs.github.com/spec/v1
-oid sha256:10c43cf3ecb8fd4f191a9de845df251285733792c198667f01c7821279943d7f
-size 690213
+oid sha256:a71c8feb1f0c7d77f497a2fd59a672ed06d49d7f28f4b11728193838e4c42601
+size 38344779
